readme.md

```diff
@@ -157,6 +187,12 @@ Type: `Function`
 
 Task function.
 
+##### skip
+
+Type: `Function`
+
+Skip function.
```

Comment:
I cover it in more detail on line 142, following the same pattern for the `task` function (which had more detail in the prose docs, and less detail in the API docs). Happy to change it, up to you 👍